File: iniciante/javascript/1064.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let countEntradas = 0
let countPositivos = 0
let somaPositivos = 0

while(countEntradas++ < 6){
    let entrada = parseFloat(lines.shift())
    if(entrada > 0){
        countPositivos++
        somaPositivos += entrada
    }
}

console.log(`${countPositivos} valores positivos`)
console.log((somaPositivos/countPositivos).toFixed(1))

File: iniciante/python/1064.py
# -*- coding: utf-8 -*-

count = 0
count_positivos = 0
entrada = []
soma = 0
media = 0.0

while count < 6:
    entrada.append(input())
    if '.' in entrada[count]:
        entrada[count] = float(entrada[count])
        if entrada[count] > 0:
            count_positivos += 1
            soma += entrada[count]
    else:
        entrada[count] = int(entrada[count])
        if entrada[count] > 0:
            count_positivos += 1
            soma += entrada[count]
    count += 1

media = soma/count_positivos

print(f'{count_positivos} valores positivos')
print(f'{media:.1f}')
